src/Components/Publishing/Display/Canvas/CanvasText.tsx

```diff
@@ -2,6 +2,7 @@ import { garamond } from "Assets/Fonts"
 import { compact, map } from "lodash"
 import React from "react"
 import styled, { StyledFunction } from "styled-components"
+import { resize } from "../../../../Utils/resizer"
```

Comment:
Can use absolute paths here (`Utils/resizer`)

Reply:
👍 Are absolute paths always preferable? I can change the rest of the import paths for these files